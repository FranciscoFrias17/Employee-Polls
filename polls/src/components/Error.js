import { useNavigate } from "react-router-dom";

function ErrorPage() {
  const navigate = useNavigate();

  const handleClick = () => {
    navigate("/");
  };

  return (
    <div text-align='center'>
      <h1>404 - Not Found</h1>
      <p>The page does not exist</p>
      <button onClick={handleClick}>Back to Home</button>
    </div>
  );
}

export default ErrorPage;
