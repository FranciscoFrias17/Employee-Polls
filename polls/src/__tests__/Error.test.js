import { render } from "@testing-library/react";
import React from "react";
import Error from "../components/Error";

describe("Error", () => {
  it("will match snapshot", () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});
