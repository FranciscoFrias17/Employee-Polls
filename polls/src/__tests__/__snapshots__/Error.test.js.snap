// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error will match snapshot 1`] = `
<div>
  <div
    class="error"
  >
    <h1>
      404 - Not Found
    </h1>
  </div>
</div>
`;
